ckanext/dcatapchharvest/profiles.py

```diff
@@ -78,7 +78,7 @@ def _add_multilang_value(
                     if values:
                         # the values can be either a multilang-dict or they are
                         # nested in another iterable (e.g. keywords)
-                        if not hasattr(values, "__iter__"):
+                        if not isinstance(values, list):
```

Comment:
Apparently in Python 3, strings also have the `__iter__` attribute. We need to really check whether the value is a list, not check for this attribute, or we won't correctly wrap a string value in a list (in the next line) and we'll end up iterating over it letter by letter.

I *think* there aren't any other kinds of iterable that we could get as a value here, only a list or a string, so this should be enough of a check here.